engine/clients/pgvector/search.py

```diff
@@ -0,0 +1,55 @@
+import multiprocessing as mp
+from typing import List, Tuple
+
+import numpy as np
+import psycopg2
+from pgvector.psycopg2 import register_vector
+from psycopg2.extras import RealDictCursor
+
+from engine.base_client.distances import Distance
+from engine.base_client.search import BaseSearcher
+from engine.clients.pgvector.config import get_db_config
+from engine.clients.pgvector.parser import PgVectorConditionParser
+
+
+class PgVectorSearcher(BaseSearcher):
+    search_params = {}
+    cursor = None
+    parser = PgVectorConditionParser()
+
+    @classmethod
+    def get_mp_start_method(cls):
```

Comment:
just to be sure:
we use this when a database client does not work with "fork", have you checked if it is indeed required to have "forkserver" or "spawn" for pgvector here?

Reply:
Will check. Thanks!

Reply:
It wasn't required. Removed it.